### PR TITLE
Notes about bash in Tips, misc changes

### PR DESCRIPTION
**Pull request by SJLennon**
Resolves codefori/vscode-ibmi#2021

### CODE REVIEW

File: src/content/docs/tips/bash.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Bash Shell Required
```
Comment (worksofliam) [inline src/content/docs/tips/bash.mdx:2]: I don't think the title should be this long. I think 'Bash Shell' is ok.

**Review by worksofliam — CHANGES_REQUESTED** [review]
@SJLennon 

A really nice guide - thank you. Please just make some minor changes:

* See my review above about the title change on your new page
* Please delete `.astro/settings.json` from the PR.

Thanks!!

**Comment by SJLennon** [comment]
@worksofliam 
Forgive my ignorance: I made the two changes and pushed to https://github.com/SJLennon/docs.  Do I just create a new pull request now? If so, what happens to the original PR?  Or do I need to do anything?

**Comment by worksofliam** [comment]
@SJLennon Nope, you did right. I can see on the PR that the commit came into the PR - and then I merged it.

That's it, until your next PR!

**Pull request merged**